fdbclient/NativeAPI.actor.cpp

```diff
@@ -1743,6 +1743,34 @@ void setNetworkOption(FDBNetworkOptions::Option option, Optional<StringRef> valu
 	}
 }
 
+// update the network busyness on a 1s cadence
+ACTOR Future<Void> monitorNetworkBusyness() {
+	state double prevTime = now();
+	loop choose {
```

Comment:
Using a `choose when` works, though it's usually intended for cases when you have multiple choices. You could instead structure it like this without the choose and when:

```
loop {
    wait(delay(...));
    // do work
}
```

Reply:
Oh okay yeah the `recurring` actor used this syntax so I was wondering if that was some convention but I can change it